matchms/filtering/derive_adduct_from_name.py

```diff
@@ -39,12 +43,12 @@ def derive_adduct_from_name(spectrum_in: SpectrumType,
     if adduct_from_name and remove_adduct_from_name:
         name_adduct_removed = " ".join([x for x in name_split if x != adduct_from_name])
         spectrum.set("compound_name", name_adduct_removed)
-        print(f"Removed adduct {adduct_from_name} from compound name.")
+        logger.info("Removed adduct %s from compound name.", adduct_from_name)
```

Comment:
It seems like some `derive_xyz` filters have logging output on success and some on failure - do we want to make this more homogeneous?